data-prepper-plugins/prometheus-sink/build.gradle

```diff
@@ -0,0 +1,27 @@
+
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation libs.armeria.core
```

Comment:
Do you need Armeria? I don't see it in use.

Reply:
Thanks for the review @dlvenable , Fixed